src/otx/cli/tools/export.py

```diff
@@ -64,12 +63,15 @@ def get_args():
         default="openvino",
     )
 
-    return parser.parse_args()
+    add_hyper_parameters_sub_parser(parser, hyper_parameters, modes=("INFERENCE",))
+    override_param = [f"params.{param[2:].split('=')[0]}" for param in params if param.startswith("--")]
```

Comment:
This looks very unreliable and requires refactoring.

Reply:
This is from other cli command file(https://github.com/openvinotoolkit/training_extensions/blob/05b8af3a6acd75806022b987c5754738159ce0f3/src/otx/cli/tools/eval.py#L71). Do you think these kind of code is unreliable or using this code in export.py is unreliable?

Reply:
In general yes, regexps with exceptions handling are more reliable, but it might be an overkill here. At least such code shouldn't be located in high-level scripts. Can we extract this part, since we already have more that one place where it's used?

Reply:
I made a function for getting override params part. The exact method for this function should be discussed with @harimkang

Reply:
I'm sure this will be removed in the future, but for now it's a pain.